### PR TITLE
increase version to v.0.13.11

### PR DESCRIPTION
**Pull request by pyu10055**
#### Description
increase version to v.0.13.11

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1389)

### CODE REVIEW

**Comment by caisq** [comment]
@dsmilkov Can you take a look at the test failure in the core-layers integration test? I've done bisecting and pinpointed the culprit to the PR: https://github.com/tensorflow/tfjs-core/pull/1379

**Comment by caisq** [comment]
@dsmilkov Here is an example test failure: 

Chrome 70.0.3538 (Mac OS X 10.13.6) Save-load round trips test-webgl1 {"WEBGL_VERSION":1,"WEBGL_CPU_FORWARD":false} Call predict() and fit() after load: conv2d model FAILED
	Failed: Error: Size(10) must match the product of shape 1,1,1568
	    at Object.inferFromImplicitShape (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:459:19)
	    at reshape_ (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:17106:18)
	    at Object.reshape (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:3341:29)
	    at Tensor.reshape (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:5247:26)
	    at Tensor.as3D (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:5164:21)
	    at MathBackendWebGL.batchMatMul (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:15130:43)
	    at environment_1.ENV.engine.runKernel.$a (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:2776:86)
	    at /tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10976:26
	    at Engine.scopedRun (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10948:23)
	    at Engine.runKernel (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10974:14)
	    at matMul_ (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:2776:40)
	    at Object.matMul (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:3341:29)
	    at Tensor.matMul (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:5289:26)
	    at Object.$b (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:2754:56)
	    at _loop_1 (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10824:47)
	    at Object.backpropagateGradients (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10842:9)
	    at /tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:11187:20
	    at /tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10938:22
	    at Engine.scopedRun (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10948:23)
	    at Engine.tidy (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10937:21)
	    at Engine.gradients (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:11176:21)
	    at Object.variableGrads (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:1985:39)
	    at SGDOptimizer.Optimizer.computeGradients (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:1232:26)
	    at SGDOptimizer.Optimizer.minimize (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:1219:23)
	    at src/engine/training.ts:1308:25 <- src/engine/training.js:1903:62
	    at src/engine/training_tensors.ts:345:23 <- src/engine/training_tensors.js:579:68
	    at /tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10938:22
	    at Engine.scopedRun (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10948:23)
	    at Engine.tidy (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:10937:21)
	    at Object.Environment.tidy (/tmp/karma-typescript-bundle-5851V8rIEJIF56Zm.js:11334:35)
	    at src/engine/training_tensors.ts:333:12 <- src/engine/training_tensors.js:564:57
	    at step (src/engine/training_tensors.js:32:22 <- src/engine/training_tensors.js:232:39)
	    at Object.next (src/engine/training_tensors.js:13:52 <- src/engine/training_tensors.js:96:32)
	    at step (src/engine/training_tensors.js:17:96 <- src/engine/training_tensors.js:117:425)
	    at Object.next (src/engine/training_tensors.js:13:52 <- src/engine/training_tensors.js:96:32)
	    at step (src/engine/training_tensors.js:17:96 <- src/engine/training_tensors.js:117:425)
	    at Object.next (src/engine/training_tensors.js:13:52 <- src/engine/training_tensors.js:96:32)
	    at fulfilled (src/engine/training_tensors.js:4:57 <- src/engine/training_tensors.js:28:44)
	    at <Jasmine>
	    at src/model_save_test.ts:212:39 <- src/model_save_test.js:647:45
